scripts/image_utils/ocr.py

```
@@ -16,7 +16,19 @@ class OCRResult:
     avg_conf: float
 
 
+def _tesseract_available() -> bool:
+    try:
+        pytesseract.get_tesseract_version()
+        return True
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)